portability-gateway/src/main/java/org/dataportabilityproject/gateway/action/createjob/CreateJobActionRequest.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I know we're not using gRPC here, but is it possible to use protos for the request and response instead of using POJO's? It probably doesn't matter in the v1, but might get rid of a lot of boiler plate.

Reply:
Yes, very good point. We'll need to investigate a transport protocol and layer if we want to support more than http.

Reply:
another option along those lines, if we don't necessarily want to support RPC but still want to simplify the request/response/action logic, is Jersey.. http://www.mkyong.com/webservices/jax-rs/jersey-hello-world-example/